doc/user/content/concepts/indexes.md

```diff
@@ -13,24 +13,37 @@ aliases:
 ## Overview
 
 In Materialize, indexes represent query results stored in memory **within a
-[cluster](/concepts/clusters/)**. You can create indexes on
-[sources](/concepts/sources/), [views](/concepts/views/#views), or [materialized
+[cluster](/concepts/clusters/)**. You can create indexes on [sources](/concepts/sources/),
+[views](/concepts/views/#views), or [materialized
 views](/concepts/views/#materialized-views).
 
+## Index structure
+
+Indexes in Materialize have the following structure for each unique row:
+
+```none
+((tuple of indexed key expression), (tuple of the row, i.e. stored columns))
```

Comment:
I copied from the create index page but wonder if it might be simpler with:

(index key, (associated rows))